app/controller/webhook/github/core.py

```diff
@@ -4,11 +4,10 @@
 import hashlib
 from db.facade import DBFacade
 from typing import Dict, Any
-from command import ResponseTuple
+from app.controller import ResponseTuple
 from config import Credentials
-from webhook.github.events.organization import OrganizationEventHandler
-from webhook.github.events.team import TeamEventHandler
-from webhook.github.events.membership import MembershipEventHandler
+from app.controller.webhook.github.events import MembershipEventHandler, \

```

Comment:
```suggestion
from events import MembershipEventHandler, \
```

I don't remember if you need to add a `.` before `events`. Not doing this is also fine, though.